.kodiak.toml

```
@@ -0,0 +1,11 @@
+version = 1
+
+[merge]
+automerge_label = "🚀 Status: ship it"
+blacklist_title_regex = "^WIP.*"
```

```suggestion
blacklist_title_regex = ".*WIP.*"
```

So that it's not necessarily how the message starts